In [1]:
import numpy as np
from scipy import stats
import matplotlib.pyplot as plt

In [2]:
fertilizer_1 = np.array([5, 6, 7, 6, 5])
fertilizer_2 = np.array([7, 8, 7, 9, 8])
fertilizer_3 = np.array([4, 5, 4, 3, 4])

In [3]:
f_value, p_value = stats.f_oneway(fertilizer_1, fertilizer_2, fertilizer_3)
print("F-value:", f_value)
print("P-value:", p_value)

F-value: 28.52631578947368
P-value: 2.7542672098048448e-05


Based on the P-value, the fertilizers don't have significantly different effects on plant growth because the P-value is not close to zero

Explain what would happen if the P-value were greater than 0.05.

As the P-value is great than 0.05 at 2.75 this means that what is happening with all three fertilizers is the same result, and they aren't all having different effects but the same effect on plant growth